### ISSUE TITLE
Add alternative text for social links

### ISSUE THREAD

**Issue by poojagunturu96** [middlebury/midd-frontend#602]
Social links use only icons and should have text alternatives for accessibility. 

Example on Siteimprove: https://my2.siteimprove.com/Inspector/7333/A11Y/Page?pageId=72160253158&impmd=D97548790CE293BDC08F6D9B84D3E607&conf=a+aa+aria+si&issue=cantTell+failed&wcag=twopointone#/sia-r11/failed/emgwZ3UwdnI=;OWJmOWFlOWE=.
